metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -202,7 +200,7 @@ public Builder setMetrics(ControllerMetrics controllerMetrics) {
 
         @SuppressWarnings("unchecked")
         public QuorumController build() throws Exception {
-            if (logManager == null) {
+            if (raftClient == null) {
                 throw new RuntimeException("You must set a metadata log manager.");
```

Comment:
Since we're now setting a raft client rather than metadata log manager, the exception message should be updated.

Reply:
Fixed.